services/payments/src/simcore_service_payments/services/webserver.py

```diff
@@ -0,0 +1,3 @@
+#
+# Placeholder to communicate back to webserver via socketio
```

Comment:
Q: probably you mean via rabbit mq?

Reply:
no, this is the call back to the webserver. Look at the diagram above. Is the upper arrow marked with socketio